learning-development/accessibility.qmd

```diff
@@ -4,33 +4,67 @@ title: "Accessibility"
 
 <p class="text-muted">Some quick links and tips for increasing awareness of accessibility in digital content</p>
 
+
+---
+
+## Accessibility in DfE
+
+DfE have our own [Digital Accessibility Hub](https://educationgovuk.sharepoint.com/sites/lvewp00043) filled with helpful information on making the digital world more accessible.
+
+There are also the [DfE design manual](https://design.education.gov.uk), which has been created by the digital teams at DfE and includes [DfE's digital accessibility guidance](https://design.education.gov.uk/accessibility).
```

Comment:
There is also that cool thing that gives you numbers of users that are likely to have specific needs which was built by someone in DfE, it is in the design manual but might be nice to mention it separately https://design.education.gov.uk/tools/how-many-users

Reply:
Ah of course, I wish I'd dug this out for the DISD weekly actually, it's a helpful one for context and helping to share the scale of users impacted! Will add in

Reply:
Made this change in my PR comments commit